.vib/geode/goss/goss.yaml

```diff
@@ -0,0 +1,54 @@
+file:
+  /bitnami/geode:
+    exists: true
+    filetype: directory
+    mode: "2775"
+    owner: root
+  /opt/bitnami/geode/config/gemfire.properties:
+    exists: true
+    filetype: file
+    mode: "0644"
+    group: root
+    contains:
+    - http-service-port={{ .Vars.locator.containerPorts.http }}
+    - jmx-manager-port={{ .Vars.locator.containerPorts.rmi }}
+command:
```

Comment:
I noticed that we are using a custom `sysctls` override setting in the `runtime_params` to disable TCP SYN cookies. According to the [official documentation](https://geode.apache.org/docs/guide/114/managing/monitor_tune/disabling_tcp_syn_cookies.html),  this is important to apply or else performance would be impacted.

I believe we should add a test to check the status of this setting, something like:

```
$ sysctl -a | grep "net.ipv4.tcp_syncookies"
```

Reply:
Good idea, added!